docs/general/installation/truenas-scale.md

```diff
@@ -0,0 +1,88 @@
+---
+uid: installation-truenas-scale
+title: TrueNAS SCALE
+description: Install on TrueNAS SCALE.
+sidebar_position: 5
+---
+
+Jellyfin is available as a [TrueNAS SCALE](https://www.truenas.com/truenas-scale/) community app. The source code for the app is available on the TrueNAS [GitHub](https://github.com/truenas/charts/tree/master/community/jellyfin).
+
+### Notes
+
+- For setup/install/network questions, try the [TrueNAS Forums](https://forums.truenas.com/).
+  - Seeking help for TrueNAS SCALE on the [Jellyfin Forums](https://forum.jellyfin.org/) will depened on active forum members having experience with SCALE.
```

Comment:
```suggestion
  - Seeking help for TrueNAS SCALE on the [Jellyfin Forums](https://forum.jellyfin.org/) will depends on active forum members having experience with SCALE.
```

Reply:
It probably would be better to change this to "depends on".